services/appveyor/appveyor-build-job.service.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const { renderBuildStatusBadge } = require('../build-status')
+const AppVeyorBase = require('./appveyor-base')
+const { NotFound } = require('..')
+
+module.exports = class AppVeyorBuildJob extends AppVeyorBase {
```

Comment:
Maybe `AppVeyorJobBuild` to match the ordering of the path?

Reply:
I suppose it'd be best to have the file name, class name, and route as aligned as possible 😄 

The last minute route base change is what caused the discrepancy here but will update the others accordingly